.vscode/settings.json

```diff
@@ -5,5 +5,8 @@
   "eslint.format.enable": true,
   "editor.defaultFormatter": "numso.prettier-standard-vscode",
   "eslint.lintTask.enable": true,
-  "wallaby.startAutomatically": true
+  "wallaby.startAutomatically": true,
+  "[vue]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
```

Comment:
did you have the `numso.prettier-standard-vscode` plugin installed, was it not working? it should cover vue code

Reply:
Not sure how my `.vscode` folder got committed. My new setup is a workspace with client and server separated - it allows me to use the mocha test explorer with both the node - commonjs - and vue - esm - syntaxes.

  I prefer eslint, but again -- this probably should have not been committed.